Doc/library/asyncio-dev.rst

```diff
@@ -148,6 +148,10 @@ adjusted::
    logging.getLogger("asyncio").setLevel(logging.WARNING)
 
 
+Network logging can block the event loop, it is recommended to use
+a separate thread for handling logs or use non blocking io.
```

Comment:
I think it should be spelled `non-blocking I/O`, but the bigger issue is that it’s not immediately clear what one should use: after all, we are in the async I/O docs!   Is there a section in the `os` module or another part of the stdlib that explains how to do non-asyncio non-blocking I/O?